README.md

```diff
@@ -223,7 +228,9 @@ one with `8786` being open. Other than that, no additional configurations should
 python runner.py --wf ttcom --executor dask/lxplus
 ```
 
-jobs automatically splitted to 100 files per jobs to avoid job failure due to crowded cluster on lxplus with the naming scheme `hist_$workflow_$json_$index.coffea`, combined in plotting scripts
+jobs automatically splitted to 50 files per jobs to avoid job failure due to crowded cluster on lxplus with the naming scheme `hist_$workflow_$json_$dictindex_$fileindex.coffea`, combined in plotting scripts.
```

Comment:
`splitted` -> `split`
`, combined in plotting scripts` -> `. The coffee files can be then combined at plotting level`